covalent_dispatcher/_cli/service.py

```diff
@@ -236,13 +244,16 @@ def _graceful_shutdown(pidfile: str) -> None:
     help="Server port number.",
 )
 @click.option("-d", "--develop", is_flag=True, help="Start the server in developer mode.")
+@click.option("--no-cluster", is_flag=True, help="Start the server without Dask")
```

Comment:
We should mention that this would default the executor to "local".

Reply:
Updated the help menu